tests/functional/sync_force_auth.js

```diff
@@ -26,68 +25,50 @@ define([
       email = TestHelpers.createEmail();
-      url = FORCE_AUTH_URL + '&email=' + encodeURIComponent(email);
-
-      client = new FxaClient(AUTH_SERVER_ROOT, {
-        xhr: nodeXMLHttpRequest.XMLHttpRequest
-      });
 
       return FunctionalHelpers.clearBrowserState(this);
     },
 
     'verified': function () {
-      var self = this;
-      return client.signUp(email, PASSWORD, { preVerified: true })
-        .then(function () {
-          return FunctionalHelpers.openPage(self, url, '#fxa-force-auth-header')
-            .execute(listenForFxaCommands)
-
-            .then(function () {
-              return FunctionalHelpers.fillOutForceAuth(self, PASSWORD);
-            })
-
-            // add a slight delay to ensure the page does not transition
-            .sleep(1000)
-
-            // the page does not transition.
-            .findByCssSelector('#fxa-force-auth-header')
-            .end()
-
-            .then(function () {
-              return FxDesktopHelpers.testIsBrowserNotifiedOfMessage(
-                          self, 'can_link_account');
-            })
-            .then(function () {
-              return FxDesktopHelpers.testIsBrowserNotifiedOfMessage(
-                          self, 'login');
-            });
-        });
+      return this.remote
+        .then(createUser(email, PASSWORD, { preVerified: true }))
+        .then(openForceAuth({ query: {
+          context: 'fx_desktop_v1',
+          email: email,
+          service: 'sync'
+        }}))
+        .execute(listenForFxaCommands)
+
+        .then(fillOutForceAuth(PASSWORD))
+
+        // add a slight delay to ensure the page does not transition
+        .sleep(1000)
+
+        // the page does not transition.
+        .then(testElementExists('#fxa-force-auth-header'))
+
+        .then(testIsBrowserNotified(this, 'can_link_account'))
+        .then(testIsBrowserNotified(this, 'login'));
     },
 
     'unverified': function () {
       var self = this;
 
-      return client.signUp(email, PASSWORD, { preVerified: false})
-        .then(function () {
-          return FunctionalHelpers.openPage(self, url, '#fxa-force-auth-header')
-            .execute(listenForFxaCommands)
-
-            .then(function () {
-              return FunctionalHelpers.fillOutForceAuth(self, PASSWORD);
-            })
-
-            .findByCssSelector('#fxa-confirm-header')
-            .end()
-
-            .then(function () {
-              return FxDesktopHelpers.testIsBrowserNotifiedOfMessage(
-                          self, 'can_link_account');
-            })
-            .then(function () {
-              return FxDesktopHelpers.testIsBrowserNotifiedOfMessage(
-                          self, 'login');
-            });
-        });
+      return this.remote
+        .then(createUser(email, PASSWORD, { preVerified: false}))
+        .then(openForceAuth({ query: {
+          context: 'fx_desktop_v1',
+          email: email,
+          service: 'sync'
```

Comment:
Could create this object in `beforeEach` just to reduce the boilerplate a little bit more.